src/chart/parameter/component/NodePropertyParameterSelect.tsx

```diff
@@ -42,94 +46,119 @@ const NodePropertyParameterSelectComponent = (props: ParameterSelectProps) => {
 
   const realValueRowIndex = props.compatibilityMode ? 0 : 1 - displayValueRowIndex;
 
+  const manualHandleParameters = () => {
+    handleParameters(paramValueTemp, paramValueDisplayTemp, false);
+  };
+  const handleParameters = (value, displayValue, manual = false) => {
+    setParamValueTemp(value);
+    setParamValueDisplayTemp(displayValue);
```

Comment:
`Local` instead of `Temp`?
In other places we call them 'input' variables, maybe we can stick to that convention.

Reply:
Done